src/coq/logicutils/hofs/hofs.ml

```diff
@@ -132,17 +132,6 @@ let map_rec_env_fix map_rec d env (sigma : evar_map) a ns ts (trm : types) =
   let d_n = List.fold_left (fun a' _ -> d a') a (range 0 n) in
   map_rec env_fix sigma d_n trm
 
-(*
- * Recurse on a mapping function with an environment for a fixpoint
- * TODO do we need both of these, or is type system too weak?
- *)
-let map_rec_env_fix_cartesian (map_rec : ('a, 'b) list_transformer_with_env) d env sigma a ns ts =
```

Comment:
What happened to this?

Reply:
It was just unused, which threw an error when running `dune build`, so I removed it

Reply:
It's used in PUMPKIN PATCH

Reply:
hmmm unless it wasn't even exposed in the interface? weird